code/menuui/playermenu.cpp

```diff
@@ -1476,7 +1476,12 @@ void player_tips_controls() {
 			Pilot.save_player(Player);
 			Pilot.save_savefile();
 		}
+
+		// popup was displayed
```

Comment:
Why not just comment above the function, either here or in the header file,
```
// returns true if the popup was displayed, false otherwise
```

Reply:
Good point